thruster_interface/scripts/thruster_interface_node.py

```diff
@@ -47,6 +47,8 @@ def __init__(self):
 
         self.output_to_zero()
 
+        rospy.on_shutdown(self.shutdown)
```

Comment:
What about just `rospy.on_shutdown(self.output_to_zero())`? I guess it's less flexible if we want to change the shutdown procedure later.